libs/hwes/event.go

```diff
@@ -143,3 +144,10 @@ func (e *Event) SetJsonData(data interface{}) error {
 func (e *Event) GetJsonData(data interface{}) error {
 	return json.Unmarshal(e.Data, data)
 }
+
+func (e *Event) Log(l *zerolog.Event) *zerolog.Event {
```

Comment:
confusing name, this method does not log anything

```suggestion
func (e *Event) GetLogEvent(l *zerolog.Event) *zerolog.Event {
```